types/redux-saga-tester/redux-saga-tester-tests.ts

```diff
@@ -1,15 +1,16 @@
 import SagaTester from 'redux-saga-tester';
+import { AnyAction, Dispatch } from 'redux'; // tslint:disable-line
 
 // constructor with all options
 new SagaTester({
     initialState: {
         orders: [],
     },
     reducers: {
-        orders: (state, action) => state,
+        orders: (state: any, action: AnyAction) => state,
```

Comment:
I don't think we should have all these `any`s here. We should let typescript infer the type based on the type definitions.